module-2/backend/week_two.md

```diff
@@ -5,21 +5,43 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+  `ActiveRecord is a object relationship manager which allows you to create relational databases with built in functions you' can call on them.`
+  
 2. What is a migration?
+  `A migration is the structure of the database which allows you to add and edit tables and ultimately allows that entire database to be recreated`
+  
 3. How does a table relate to a model?
+  `A model communicates with the table in a database, the model contains methods which can be called on the database and allows us to draw relationships between tables`
+  
 4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+  `They are class methods, they are called on the class and not on instances of the class`
```

Comment:
Tricky question, but these are instance methods. The important take away is that you'll need a specific instance of a class before you can use the functionality that either `belongs_to` or `has_many` offers you. E.g.:

```ruby
blue_car = Car.find(1) #finds the car with an id of 1.
blue_dealership = blue_car.dealership # finds the dealership for that *specific* car (the instance of a Car) assuming that we have `belongs_to :dealership` in our Car model.
```